pkg/controllers/testutil/zcmd/zfs/get.go

```diff
@@ -0,0 +1,50 @@
+/*
+Copyright 2020 The OpenEBS Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package zfs
+
+import (
+	"fmt"
+	"strings"
+)
+
+// GetProperty mocks the zfs get command and returns the error based on the output
+func (mPoolInfo *MockVolumeInfo) GetProperty(cmd string) ([]byte, error) {
```

Comment:
This code no where returns error.

Reply:
mPoolInfo --> Should it be mVolInfo

Reply:
And then `mPoolInfo` is nowhere used. 
Worst case -- let us put a comment indicating future use.

Reply:
Added the comments and updated